src/TopPage/FluentExtension.php

```diff
@@ -35,6 +35,37 @@ public function updateCMSFields(FieldList $fields)
         $fields->removeByName('TopPageLocale');
     }
 
+    /**
+     * Covers the case where top page metadata is populated during duplication process of localised data
```

Comment:
Prob be good to add a bit more to this function about why it exists yeah?

Its likely the **only** mention of `Locale` in this entire codebase right?
Perhaps a note of this is to help compatibility with Fluent, a link to the fluent repo, etc...

🤔

Reply:
There are already docs related to this extension and what's the purpose of it.

Reply:
Since this is explicitly the `FluentExtension` I think it's clear enough that this pertains to logic in `tractorcow/silverstripe-fluent` as mentioned in the PHPDoc for the class itself.